### DISCUSSION TITLE
Moes BRT-100-TRV Max_temperature

### DISCUSSION THREAD

**Discussion by Kup9l777** [Koenkk/zigbee2mqtt#10571] [Devices]
Good day! please change the ability to set the maximum temperature to 45. Now it is listed as 5 - 35. The device itself has the ability to set up to 45! Plus in Tuya it is possible to set up to 45, but the instructions for the device are an error ... Because of this, I have a lot of errors in the logs.  You need to change two parameters max_temperature and current_heating_setpoint - 45
[https://ltdfoto.ru/image/EYgppw](url)
[https://ltdfoto.ru/image/EYg9W8](url)

**Comment by Koenkk** [comment]
This has already been fixed (https://github.com/Koenkk/zigbee2mqtt/issues/10564)

**Reply by Kup9l777** [reply]
> This has already been fixed (#10564)

I updated zigbee-herdsman-converters but the error still persists

**Reply by Kup9l777** [reply]
kbr@raspberrypi:~ $ npm run lint
npm ERR! missing script: lint

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/---/.npm/_logs/2022-01-09T04_35_42_324Z-debug.log
kbr@raspberrypi:~ $ npm test
npm ERR! missing script: test

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/---/.npm/_logs/2022-01-09T04_36_01_829Z-debug.log

**Reply by Kup9l777** [reply]
npm i zigbee-herdsman-converters - this command was successful
